migrations/6_initialize_721token.js

```diff
@@ -46,6 +46,26 @@ module.exports = async (deployer, network, accounts) => {
       await proxiedCodexRecord.setStakeContract(
         CodexStakeContract.address
       )
+
+      if (network === 'ganache') {
+        const codexRecordAddress = proxiedCodexRecord.address
+        const codexCoin = await CodexCoin.deployed()
+        const faucetAccount = accounts[2]
+
+        // So that the faucet account can mint tokens for giveaways
+        await codexCoin.approve(codexRecordAddress, web3.toWei(100000, 'ether'), { from: faucetAccount })
+
+        /* eslint-disable no-await-in-loop */
+        for (let i = 0; i < accounts.length; i++) {
+
+          // arbitrary approval amount, should be sufficient for the purposes of local dev
+          await codexCoin.approve(codexRecordAddress, web3.toWei(100000, 'ether'), { from: accounts[i] })
```

Comment:
Isn't this duplicating the work on line 56 (since `faucetAccount === accounts[2]`)? I think lines 55 & 56 can be removed, but maybe I'm missing something.

Reply:
yep you're right